base/file.jl

```diff
@@ -29,6 +30,8 @@ export
     unlink,
     walkdir
 
+import Base.==
```

Comment:
Why...? Just `Base.==` like the other functions in this file that are getting extended?

Reply:
It was the only one that I couldn't just add a method via fully qualifying the function name. An error said this was needed. I might have misunderstood?

Reply:
Did you use `Base.:(==)(a, b) = ...`